maestro-client/src/main/java/maestro/utils/TapUtils.kt

```diff
@@ -0,0 +1,73 @@
+package maestro.utils
+
+import com.github.romankh3.image.comparison.ImageComparison
+import maestro.Driver
+import maestro.Point
+import maestro.ViewHierarchy
+import org.slf4j.LoggerFactory
+import java.awt.image.BufferedImage
+
+class TapUtils {
```

Comment:
Util classes such as this one (and actually `ScreenshotUtils` too) seem to break encapsulation and spread business logic across the codebase. Ideally, `Maestro` class should serve that role. Can we find a way to contain all that branching logic there instead?